src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -454,6 +454,10 @@ public void ShutDown(bool shutdownHost)
 
             AnalyticsService.ShutDown();
 
+            LuceneSearch.LuceneUtilityNodeSearch = null;
```

Comment:
so what were we leaking?

Reply:
LuceneUtilityNodeSearch holds a reference to the DynamoModel.
So the DynamoModel was not being garbage collected because it still had refs.